app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,10 @@
+class SessionsController < ActionController::Base
+
+  def index
```

Comment:
What's the `index` method for?  If it's the page that triggers the `#create` JS, shouldn't it be `new`?